### DISCUSSION TITLE
jakartify sardine

### DISCUSSION THREAD

**Discussion by MichaelEnke** [lookfirst/sardine#357] [General]
Hi there,
I migrate a project which uses Sardine from Tomcat 9 to Tomcat 10. Since Tomcat changed from javax (9 and before) to the jakarta import namespace, all depending libraries in this project need to be updated to the "jakartified" version. For Sardine there seems to be no jakartified version. Would that be a good reason to create a version 6.0?
Thanks.

**Comment by MichaelEnke** [comment]
[sardine-jakarta-5.10-sources.patch](https://github.com/lookfirst/sardine/files/10824279/sardine-jakarta-5.10-sources.patch)
Here comes a patch for sardine-5.10-sources to exchange javax.xml.bind. with jakarta.xml.bind.

**Reply by lookfirst** [reply]
Hi @MichaelEnke. Thanks for the patch... any way that we can get you to submit it as a PR? Much easier to view on the site here and merge it.

**Reply by MichaelEnke** [reply]
I never did this before on github - I guess I have to create a branch first? Then I need write permission on the repo?

**Reply by lookfirst** [reply]
Hmm... I really hate to be that guy and respond to you with a 'let me google that for you'.... but I'm really not sure how else to get you started on things. https://www.google.com/search?q=how+to+do+a+pull+request+on+github

If you're really not interested in going through all those documents and learning the git/github workflow process (I fully understand), I looked at your patch and it is literally just find/replace on the imports, which is something we can take care of for you.

**Reply by MichaelEnke** [reply]
I'd appreciate if you take care of for me. I read that I would have to fork etc.
